SweetPea to English/gpt3prompt_derived_factors.txt

```diff
@@ -282,3 +284,118 @@ Text:
 The color response factor depends on the color factor and has two levels: -1 and 1. If the color is red or blue, then the corresponding color response is -1 or 1, respectively.
 
 Code:
+col      = factor("Color",  [Level("red",2), "green", "blue", "brown"])
+wrd       = factor("Word", ["red", "green", "blue", "brown"])
+
+def is_response_left(col_factor):
+    return col_factor == "red"
+def is_response_right(col_factor):
+    return col_factor == "green"
+def is_response_up(col_factor):
+    return col_factor == "blue"
+def is_response_down(col_factor):
+    return col_factor == "brown"
+
+resp = factor("response", [
+    derived_level("left", within_trial(is_response_left, [col]),2),
+    derived_level("right", within_trial(is_response_right, [col])),
+    derived_level("up", within_trial(is_response_up, [col])),
+    derived_level("down", within_trial(is_response_down, [col]))
+])
+
+Text:
+The response factor depends on the Color factor and has four levels: left, right, up, and down. If the Color is red, green, blue, or brown, then the corresponding response is left, right, up, and down, respectively. The response left occurs twice as often as the response right, up or down.
+
+Code:
+color      = factor("Color",  ["red", "green", "blue", "brown"])
+word       = factor("Word", ["red", "green", "blue", "brown"])
+
+def is_congruent(color, word):
+    return color == word
+
+def is_incongruent(color, word):
+    return not is_congruent(color, word)
+
+congr_level = derived_level("congruent", within_trial(is_congruent, [color, word]),weight=4)
+incongr_level = derived_level("incongruent", within_trial(is_incongruent, [color, word]))
+
+congr_factor = factor("congruency", [
+    congr_level,
+    incongr_level
+])
+
+Text:
+The congruency factor depends on the regular factors color and word and has two levels: congruent and incongruent. A trial is considered congruent if the color matches the word, otherwise, it is considered incongruent. In our experiment the frequency between congruent and incongruent trials is different. There are four times as many congruent trials then incongruent trials.
+
+Code:
+num      = factor("digit",  [1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+color      = factor("Color",  ["red", "green", "blue", "brown"])
+word       = factor("Word", ["red", "green", "blue", "brown"])
+
+def is_e(num):
+    return (int(num) % 2) == 0
+
+def is_o(num):
+    return not is_e(num)
+
+o_level = derived_level("odd", within_trial(is_e, [num]),3)
+e_level = derived_level("even", within_trial(is_o, [num]),2)
+
+par_factor = factor("parity", [
+    o_level,
+    e_level
+])
+
+Text:
+The parity factor depends on the regular factor digit has two levels: odd and even. A trial is considered even if the digit is divisible by 2, otherwise, it is considered odd. In our experiment in the factor parity the ratio between odd and even is 3 to 2.
```

Comment:
```suggestion
The parity factor depends on the regular factor digit has two levels: odd and even. A trial is considered even if the digit is divisible by 2, otherwise, it is considered odd. The experiment sequences were generated such that the ratio between odd and even trials was 3 to 2.
```